File: .skills/openclaw-skills/skills/monkey-company/leproxyfrancais/examples/socks5.py
"""Le Proxy Français — Proxy SOCKS5 (Mutualisé ou Dédié)"""
import os
import requests

API_KEY = os.environ["LPF_API_KEY"]

# Mutualisé (3 crédits/Go) — changer en ded.prx.lv:1081 pour Dédié (8 crédits/Go)
proxies = {"https": f"socks5h://{API_KEY}@mut.prx.lv:1080"}

res = requests.get("https://ipinfo.io/ip", proxies=proxies)
print(f"IP: {res.text.strip()}")

File: .skills/openclaw-skills/skills/monkey-company/leproxyfrancais/examples/socks5.js
/** Le Proxy Français — Proxy SOCKS5 (Mutualisé ou Dédié) */
const { SocksProxyAgent } = require("socks-proxy-agent");

// Mutualisé (3 crédits/Go) — changer en ded.prx.lv:1081 pour Dédié (8 crédits/Go)
const agent = new SocksProxyAgent(
  `socks5h://${process.env.LPF_API_KEY}@mut.prx.lv:1080`
);

fetch("https://ipinfo.io/ip", { agent }).then(async (res) => {
  console.log(`IP: ${await res.text()}`);
});
